src/components/Editor.tsx

```diff
@@ -27,6 +27,7 @@ export const Editor = ({
     appSessionStateAccessor,
   } = useAppServices();
 
+  const userFieldsQuery = useGetUserFields();
```

Comment:
I was taking a screenshot to illustrate the PR when I saw this:

![image](https://user-images.githubusercontent.com/1157864/152395491-fd81bed2-b223-4be3-a1dd-366127a60dd9.png)

Probably we have a bug here: the `getFields` request should be done after finishing the `getUserData` request because the JWT token comes there.

Checking...

Reply:
Fixed in #94

Reply:
Fixed! 

![image](https://user-images.githubusercontent.com/1157864/152543700-deba95c4-1923-409f-a089-3af9def12f5e.png)